HelloWorld/force-app/main/default/lwc/helloWorld/helloWorld.js-meta.xml

```diff
@@ -1,5 +1,11 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <LightningComponentBundle xmlns="http://soap.sforce.com/2006/04/metadata">
-    <apiVersion>49.0</apiVersion>
-    <isExposed>false</isExposed>
+    <apiVersion>48.0</apiVersion>
```

Comment:
Technically didn't need to lower this version number but since most trailhead LWC examples use v48 I decided to use v48 as well.